ush/narre/ush_narre_plot_py/change.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
```

Comment:
@BinbinZhou-NOAA I don't think this file is needed?  Looks like just a script you ran to make changes in other files.

Reply:
@BinbinZhou-NOAA - please respond so we can get this PR cleaned up and good to merge.